Firestore/core/src/firebase/firestore/model/model_fwd.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_MODEL_MODEL_FWD_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_MODEL_MODEL_FWD_H_
+
+#include <cstdint>
+#include <unordered_map>
+
+#include "absl/types/optional.h"
+
+namespace firebase {
+
+class Timestamp;
+
+namespace firestore {
+
+class GeoPoint;
+
+namespace util {
+
+struct Empty;
```

Comment:
Is this one used?

Reply:
No, sorry. This is leftover from when SortedSet had the much more complicated type signature. I should have cleaned this up.